Find a functionally equivalent code:android.view.HardwareLayer.copyInto:COMMENT
<android.view.HardwareLayer: boolean copyInto(Bitmap)>
Method Modifier: public      default     abstract    
Method Modifier: No          Yes         Yes         
Method Updated:  Yes         No          No          
Comment:/**
 * Copies this layer into the specified bitmap.
 *
 * @param bitmap The bitmap to copy they layer into
 *
 * @return True if the copy was successful, false otherwise
 */

@@ -1 +1,3 @@
-
+{
+    return mRenderer.copyLayerInto(this, bitmap);
+}

